lib/diagnostic.js

```diff
@@ -6,26 +6,38 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (stringOfWords) {
+  return stringOfWords.replace(/\W+/,'').toUpperCase();
```

Comment:
You were on the right track but you need to split the string into an array by `stringOfWords.split(' ');`. Then once you have the array, you need to iterate through each element in the array applying the `splitWords.replace(/\W+/,'').toUpperCase();` to each item of the array.